chapters/Basics.md

```diff
@@ -217,7 +217,10 @@ console.log("Hello world!");
 
 In this case, there was very little for TypeScript to transform, so it looks identical to what we wrote.
 The compiler tries to emit clean readable code that looks like something a person would write.
-While that's not always so easy, TypeScript indents consistently, is mindful of when our code spans across different lines of code, and tries to keep comments around.
+While that's not always so easy, TypeScript:
```

Comment:
Yea, so I proposed bullets because it seemed hard to read as one long sentence. It visually helped when I separated them as a bulleted list. Could just be a me thing!